docs/notebooks/lunar_lander_videos/generate_video.py

```diff
@@ -0,0 +1,97 @@
+"""This script is used to generate videos for the OpenAI Gym notebook.
+
+First two functions, as well as constants, shall be in sync with the notebook.
+At the bottom of this file there are parameters and random seeds used to generate each video.
+The video and several json files will be created in this folder, with some auto-generated names.
+
+Depending on your environment, there might be various dependecies you'd need to run this script.
+In general, these may include:
+apt install python-opengl
+apt install ffmpeg
+apt install xvfb
+apt install x111-utils
+pip install pyglet
+
```

Comment:
Should we add these to the notebook requirements.txt?

Reply:
Short answer: no, we shouldn't.

Long answer. These aren't required to run the notebook. People would need them if they want to generate their own videos, but I don't have a clue if this list is complete. In my env (Ubuntu WSL) it was enough, but perhaps Mac OS users will need a different setup. Besides, most of these aren't python libs, these are linux packages.